docs/application/native/guides/device/battery-monitor.md

```diff
@@ -39,37 +41,33 @@ To enable your application to use the Battery Monitor functionality:
 
 To get the battery usage information related to the application ID:
 
-- Get battery usage information of an application for a particular resource over a specific duration of time.
-  To get the resource ID, use the `battery_monitor_resource_id_e` enum and to get the time duration type information, use the `battery_monitor_duration_type_e` enum:
+- Get the battery usage information of an application for a particular resource over a specific interval of time.
+  To get the resource ID, use the `battery_monitor_resource_id_e` enum. The timestamps **from** and **to** must be set as per the Unix Epoch time format:
 
     ```
     #include <battery_monitor.h>
+    #include <time.h>
 
     void
     func(void)
     {
-       char* app_id = "org.tizen.samsung";
-
        int error_code = BATTERY_MONITOR_ERROR_NONE;
-       int battery_usage = -1;
-       battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
-       battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
 
-       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
-
-       if (error_code == BATTERY_MONITOR_ERROR_NONE)
-           printf("The Battery Usage for appid [%s], for resource display is [%d]", app_id, battery_usage);
-       else
-           printf("Error Occurred [%d]", error_code);
+       char* app_id = "org.tizen.samsung";
+       double battery_usage = -1;
+       battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_CPU;
+       time_t now, from = -1, to = -1; time(&now);
 
-       resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
-       duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+       /* To get the usage for past thousand seconds from current epoch time */
+       int period_t = 1000;
+       to = now; from = to - period_t;
 
-       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+       error_code = battery_monitor_get_power_usage_by_app_per_resource(app_id, resource_id, from, to, &battery_usage);
 
        if (error_code == BATTERY_MONITOR_ERROR_NONE)
-           printf("The Battery Usage for appid [%s], for resource wifi is [%d]", app_id, battery_usage);
-
+           printf("The Battery Usage for appid [%s], for resource cpu is [%lf] mAh", app_id, battery_usage);
```

Comment:
@nawab-samsung Question. According to your other comments, isn't "by" better than "for"?

The Battery Usage _by_ appid [%s],...

Reply:
@mijin85cho  I think appId is a value(a reference to an app/ index of an app),  but not the app itself, so it( the value itself) may not consume battery that why for is appropriate. My understanding can be wrong, so kindly give your valuable suggestion, will go as you say.

I do agree that "by" gives better clarity but it is situation based, 
Also you had commented with your suggestion https://github.com/Samsung/tizen-docs/pull/1076#discussion_r409361830. 
So kept it intact. 
@d1-agarwal What are your thoughts on this?

Reply:
@mijin85cho I did understand the capitalization of B of battery and U of usage, but here in the comment, you wrote The Battery Usage _by_ appid [%s],...
here in [comment](https://github.com/Samsung/tizen-docs/pull/1076/files#r413460815)

But in [comment](https://github.com/Samsung/tizen-docs/pull/1076#discussion_r409361830) you kept _for_ but capitalized B & U 

So that was contradictory for me. 

Now with https://github.com/Samsung/tizen-docs/pull/1076/files#r414343175 it is clear that we dont need to change the sentences as for relates better for this situation. Just focus on  capitalization.